pkg/pillar/types/volumes.go

```diff
@@ -195,6 +195,11 @@ type VolumeStatus struct {
 	RefCount  uint
 	LastUse   time.Time // When RefCount dropped to zero
 	PreReboot bool      // Was volume last use prior to device reboot?
+
+	ReservedSpace uint64 // Contribution to global ReservedSpace
+	Size          uint64 // Once DOWNLOADED; in bytes
```

Comment:
Is the invariant that exactly one of these are non-zero?

Reply:
I don't know, to be honest. I took these "as-is" from downloader status, not really trying to understand them.